src/Game.js

```diff
@@ -0,0 +1,40 @@
+const GAME_ID = 'EtQZjbZctTfC60wlEG9f';
```

Comment:
Please configure code linter from [this](https://github.com/microverseinc/linters-config/tree/master/javascript) config and make sure all the checks are passing on PR.

Reply:
I have added linter correctly, but I don't know why they are not working